src/items/powerup_manager.cpp

```diff
@@ -219,7 +216,7 @@ void PowerupManager::WeightsData::reset()
 void PowerupManager::WeightsData::readData(int num_karts, const XMLNode *node)
 {
     m_num_karts = num_karts;
-    for (unsigned int i = 0; i < node->getNumNodes(); i++)
+    for (unsigned int i = 0; i < node->getNumNodes(); ++i)
```

Comment:
Leave i++ in loops. The vast, vast majority of loops in the STK code base use post-increment notation, so this change creates inconsistencies.

In for loops, both pre-increment and post-increment will generate exactly the same code, so this is purely a stylistic matter.